### PR TITLE
fix cube root and log

### PR DESCRIPTION
**Pull request by Xdminsy**
fix some errors in common operation description

### CODE REVIEW

**Comment by joe11051105** [comment]
請使用繁體中文，感謝！

**Comment by Xdminsy** [comment]
(⊙o⊙)哦.

File: basic_operation/common_operation.md [hunk 1]
```diff
@@ -72,10 +72,10 @@
 ### log
 
 ```r
-> log(10) # log 以 1 為底
+> log(10) # log 以 e 為底
 [1] 2.302585
 
-> log1p(9) # log(x) = loglp(1+x)
+> log1p(9) # log(x) = log1p(x - 1)
```
Comment (joe11051105) [inline basic_operation/common_operation.md]: log(x) = loglp(1+x) 並沒有錯

ex：log(9) = log1p(8)

**Comment by Xdminsy** [comment]
But In log(9), the x is 9, so log1p(1 + x) is log1p(10) not log1p(8)
你可以試試f1 <- function(x){isTRUE(all.equal(log(x), log1p(1 +x )))}和f2 <- function(x){isTRUE(all.equal(log(x), log1p(x - 1)))}

**Comment by joe11051105** [comment]
sorry，我看錯了，感謝。

**Pull request merged**